karapace/backup/backends/v2.py

```diff
@@ -60,8 +78,10 @@ def serialize_record(
 
 
 class SchemaBackupV2Reader(BaseItemsBackupReader):
+    marker_size: ClassVar = 4
+
     @staticmethod
-    def items_from_file(fp: IO[str]) -> Generator[tuple[str, str], None, None]:
```

Comment:
This is so that V1 and V2 fits into the same interface. For V1 I think we used to lie and use `tuple[str, str]` as type while it could never actually be that at runtime because it returns whatever `json.loads()` reads.

I _believe_ mypy didn't accept `tuple[str, str]` once the interface stipulates `Sequence[str]`.